### ISSUE TITLE
Warning: Unknown props

### ISSUE THREAD

**Issue by moust** [tommoor/emojione-picker#28]
I have this warning threw by React :
```
Warning: Unknown props `unicode_alternates`, `category`, `emoji_order`, `aliases`, `aliases_ascii`, `keywords` on <div> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in div (created by Emoji)
    in Emoji (created by Picker)
    in li (created by Picker)
    in ul (created by Picker)
    in div (created by Picker)
```
I use React 15.2.1

**Comment by yofret** [comment]
I'm having the same Issue, with multiple props

**Comment by tommoor** [comment]
Thanks! This is fixed in 0.4.0, just released to npm
